Draft guidance on dubbed videos (#1631)
w3c/wai-website: pages/about/translating/resources/dubbed-videos.md

---
title: "How to Provide a Dubbed Version of a WAI Video"
nav_title: Dubbed Videos
lang: en
last_updated: 2025-11-05
description: Help make the Web accessible to people with disabilities around the world. We appreciate your contributions to translating W3C WAI accessibility resources.

permalink: /about/translating/resources/dubbed-videos/
ref: /about/translating/resources/dubbed-videos/

github:
  label: wai-translation-guidance

image: /content-images/about/social-translations.png
feedbackmail: wai@w3.org

footer: |

>   <p><strong>Editors:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a> and Rémi Bétin.</p>

Reviewer: ```suggestion
  <p><strong>Editors:</strong> Rémi Bétin and <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
```

---

{::nomarkdown}
{% include box.html type="start" h="2" title="Summary" class="full" %}
{:/}

This page guides you through the steps to provide a dubbed version of a video on the WAI website.

{::nomarkdown}
{% include box.html type="end" %}
{:/}

{::options toc_levels="2" /}
{::nomarkdown}
{% include_cached toc.html type="start" title="Page Contents" class="simple" %}
{:/}

-   TOC is created automatically.
{:toc}

{::nomarkdown}
{% include toc.html type="end" %}
{:/}

## Overview

> 

Author: +1. I've added an ‘Introduction’ section to address this point.

There are 4 steps to provide a dubbed version of a WAI video:

1. [Request permission](#permission)
2. [Provide subtitles, or use existing ones](#subtitles)
3. [Dub the video](#dubbing)
4. [Deliver the final video to W3C](#review)

## Step 1: Request permission {#permission}

Before starting the production of a dubbed video, you must ask and receive permission from W3C. Please email your request to <group-wai-translations@w3.org>, or open a new issue [in the `wai-translations` GitHub repository  {% include_cached external.html %}](https://github.com/w3c/wai-translations/).

> In your message, state that you agree to contribute under the [Translation agreement](https://www.w3.org/WAI/about/translating/#translation-agreement). 

Reviewer: that includes "Crediting" information which we will probably do differently for videos, since they are mostly likely done by and organization rather than an individual. Maybe put relevant info here instead of pointing them over there?

> Some videos have a separate version with audio description – for example, the [Perspective Videos](https://www.w3.org/WAI/perspective-videos/). In such cases, we will only accept intents to translate _both_ versions of the video.

Author: New version: 

> In such cases, we will only accept intents to dub _both_ versions of the video

## Step 2: Provide subtitles, or use existing ones {#subtitles}

- If the video does not have subtitles in your language, please provide a text translation of the English captions/subtitles file. See the [workflow for creating video subtitles](/about/translating/resources/subtitles/).
- Some videos already have subtitles. If you would like to update the existing subtitles, or if you would like to change the wording to be smoother in your language, please request approval via e-mail <group-wai-translations@w3.org>.

In any case, please submit your script for approval by the W3C team before recording.

## Step 3: Dub the video {#dubbing}

The dubbed version must include the entire original video, including the ending with the link to w3.org/WAI.

> Any changes to the video must be first approved by the W3C team.

Reviewer: Stronger, maybe: We generally not not allow changes to the videos. If you have suggestions for changes based on the translation, they must be first approved...

Author: Suggestion applied.

## Step 4: Deliver the final video to W3C {#review}

When the dubbed video is ready, provide it to <group-wai-translations@w3.org> for final review and publication. 

> Please note that W3C retains copyright to the video, and may publish the video on their website, YouTube channel and other places. The video can be redistributed by you and others in accordance with the License terms.

Reviewer: ```suggestion
Please note that W3C retains copyright to the video, and may publish the video on out website, YouTube channel, and other places. The video can be used by you and others following the License terms.
```
